### PR TITLE
Removed the close button from the top-right of the alert dashboard flyout. Refactored monitor details page for anomaly detection monitors.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
Signed-off-by: Thomas Hurney [hurneyt@amazon.com](mailto:hurneyt@amazon.com)

### Description
1. Refactored the alert dashboard flyout to remove the close button in the top-right corner. 
2. Moved the detector link on the monitor details page for anomaly detection monitors from under the monitor name to the overview table. Other trigger definition types will not display that field.

### Screenshots
Screen Shot 2021-08-19 at 10 32 57 AM

Screen Shot 2021-08-19 at 11 02 53 AM

Screen Shot 2021-08-19 at 11 02 43 AM

 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/pages/MonitorDetails/components/MonitorOverview/utils/getOverviewStats.js [hunk 1]
```diff
@@ -56,7 +58,7 @@ function getMonitorType(searchType) {
 function getMonitorLevelType(monitorType) {
   console.log(monitorType);
```
Comment (leeyun-amzn) [inline public/pages/MonitorDetails/components/MonitorOverview/utils/getOverviewStats.js:59]: Just saw this. Do we want to remove this `console.log` line?

**Review by leeyun-amzn — APPROVED** [review]

File: public/pages/MonitorDetails/components/MonitorOverview/utils/getOverviewStats.js (hunk 1 above)
Comment (AWSHurneyt) [reply public/pages/MonitorDetails/components/MonitorOverview/utils/getOverviewStats.js:59]: Good catch. I was actually just about to ask about this too. I think it'd be worthwhile to move the log to the `default` case, and reword it to something like "Unexpected monitor type".

Comment (AWSHurneyt) [reply public/pages/MonitorDetails/components/MonitorOverview/utils/getOverviewStats.js:59]: Per offline conversation, we'll resolve this in a separate PR by moving the `log` line to the `default` case, and will investigate possibly adding a toast popup to that case as well in the future.
